packages/core/src/analytics/integrations/__tests__/Omnitracking.test.js

```diff
@@ -372,7 +373,96 @@ describe('Omnitracking', () => {
           expect(mockLoggerWarn).not.toHaveBeenCalled();
         });
       });
+
+      describe('Event Tracking', () => {
+        describe('Interact Content', () => {
+          it('should warn if an interact content does not have required parameters', async () => {
+            const data = generateTrackMockData({
+              event: eventTypes.INTERACT_CONTENT,
+              properties: {
+                interactionType: undefined,
+              },
+            });
+            await omnitracking.track(data);
+
+            expect(mockLoggerWarn).toHaveBeenCalledWith(
```

Comment:
I would also add an `expect` to make sure that the event is not being sent to omnitracking in this case. If you do that, change the test description to something like this: "should not track an interact content event if the required parameters are missing" or something like that.